README.md

```diff
@@ -0,0 +1,219 @@
+    - [Balancing life and work](#balancing-life-and-work)
+- [Communicating With Me](#communicating-with-me)
+    - [Talking](#talking)
+    - [Slack](#slack)
+    - [Email](#email)
+    - [PagerDuty](#pagerduty)
+    - [Mobile phone](#mobile-phone)
+    - [Social media](#social-media)
+- [Bugs & Quirks](#bugs--quirks)
+    - [Direct and intense](#direct-and-intense)
+    - [Fast talker, sporadic rambler](#fast-talker-sporadic-rambler)
+    - [Stubborn](#stubborn)
+    - [Gets bored easily](#gets-bored-easily)
+- [My Story](#my-story)
+    - [Chapter 1: NOVA](#chapter-1-nova)
+    - [Chapter 2: State College](#chapter-2-state-college)
+    - [Chapter 3: Various](#chapter-3-various)
+    - [Chapter 4: San Francisco](#chapter-4-san-francisco)
+- [Personal Interests](#personal-interests)
+- [Want to work together?](#want-to-work-together)
+
+<!-- /TOC -->
+
+## Management Philosophy
+
+The following are the core tenets that provide the foundation for how I prefer to manage. I strive to adhere to these all the time. If you ever find me to be in violation of one of these ideas, please point it out to me. That will help me do better in the future.
+
+- __Measuring stick for success:__ Can someone go out and get a better role after you’ve been their manager? You’ve got to care about the development of people to your core.
+- __Put people and the team first:__ Constantly ask what you can do to be most helpful to individuals or the team? The most helpful thing you can do may be totally different day-to-day, week-to-week, person-to-person, or team-to-team. Default to the assumption that you’re the worst on the team and treat everyone else like they’re more awesome than you. If you’ve hired right, that will be true.
+- __Constant improvement:__ Make this a fundamental value for both individuals and the team itself. Don’t set anything in stone. Don’t assume you’re ever doing something perfect. When you’ve made one improvement, move onto the next thing. Repeat forever.
+- __How to move mountains:__ Give a properly motivated team a high degree of trust and autonomy.
+- __Achieving motivation:__ Help everyone to understand the value of what the team is doing and what they as an individual are doing. Make sure everyone knows the priorities and how they were determined. Be transparent and honest about goals. Keep your reports' growth central to your focus and make it a part of their everyday life.
+- __Autonomy:__ Let teams decide for themselves how they best operate. Actively seek out new ideas. When someone has an idea, consider it with an open mind and err on the side of trying new things. You can always revert if something doesn’t work. Don’t micromanage someone’s day or their work.
+- __Create a safe and comfortable psychological space__: Strive to craft this work environment for engineers and teams. Encourage people to share their ideas. Listen to those opinions and feedback. Take action on what you’re hearing. Foster a culture of blameless criticism.
```

Comment:
I don't believe the concepts to be mutually exclusive. An entity can have ownership while also practicing blameless criticism.

A hypothetical example: you write a bug that causes an emergency. You take ownership by fixing the bug and driving the retrospective. The group practices blameless criticism by focusing on the context that resulted in the issue occurring in the first place rather than who the individual was that caused the mistake.